test/acceptance/features/enrich_payment_request.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Enrich Payment Request
+
+  Scenario: Enrich Request
+    Given a valid payment request is received containing details
+      | dueDate     | 2022-12-01                 |
+      | description | G00 - Gross value of claim |
+      | accountCode | SOS273                     |
```

Comment:
I'd maybe leave the account code out of the test as we don't need it.  We'll determine what the account should be in the processing service, rather than depending on whether we already have one